PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -341,11 +386,49 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
     return rejectionMask;
   }
 
+  /// \brief Applies event selection with upc flag and bc para
+  /// \tparam useEvSel use information from the EvSel table
+  /// \tparam centEstimator centrality estimator
+  /// \param collision collision to test against the selection criteria
+  /// \param centrality collision centrality variable to be set in this function
+  /// \param ccdb ccdb service needed to retrieve the needed info for zorro
+  /// \param registry reference to the histogram registry needed for zorro
+  /// \return bitmask with the event selection criteria not satisfied by the collision
+  template <bool useEvSel, o2::hf_centrality::CentralityEstimator centEstimator, typename BCs, typename Coll>
+  uint32_t getHfCollisionRejectionMask(const Coll& collision, float& centrality, o2::framework::Service<o2::ccdb::BasicCCDBManager> const& ccdb, o2::framework::HistogramRegistry& registry, const BCs& bcs)
```

Comment:
What's the benefit of having two functions?

Reply:
> Templetisation should be enough (see my comment above)

No, there is a parameter for bcs, we have to load it in the UPC selection.

Reply:
> If you use a pointer which is `nullptr` by default, you only need one function.

Thanks @vkucera, which should be the best solution!